src/url.js

```diff
@@ -37,12 +38,15 @@ function recursiveEncode(input) {
 
 var refinementsParameters = ['dFR', 'fR', 'nR', 'hFR', 'tR'];
 var stateKeys = shortener.ENCODED_PARAMETERS;
-function sortQueryStringValues(prefixRegexp, a, b) {
+function sortQueryStringValues(prefixRegexp, new2Old, a, b) {
```

Comment:
first thing I notice if coming to the code is "what is new?" "what is old?"

what about "userMapping" or "customMapping"?